bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -640,13 +641,29 @@ def payment_success(self):
+            )
+            self.br.execute_script("arguments[0].click();", download_btn)
+            filename, content = self.wait_for_download(expected_filename="transfer.pdf")
+            if filename and content:
+                save_file(
+                    self.docname + " Payment Receipt.pdf",
+                    content,
+                    self.doctype,
+                    self.docname,
+                    is_private=1,
+                )
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)